elastic/kibana
# [Fleet] Support installing bundled package without providing version

## Summary

Resolve https://github.com/elastic/kibana/issues/161270

Allow to install bundled packages without providing version 

## How to tests

Configure a bundled APM packages by configuring your bundled package location and adding an APM package there
```
xpack.fleet.developer.bundledPackageLocation: /tmp/test-packages
```

Try to install APM it should work 
```
POST /api/fleet/epm/packages/apm
```
